socorro/external/postgresql/adi.py

```diff
@@ -0,0 +1,84 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import datetime
+import logging
+
+from socorro.external.postgresql.base import PostgreSQLBase
+from socorro.external import MissingArgumentError
+from socorro.lib import external_common
+
+logger = logging.getLogger("webapi")
+
+
+class ADI(PostgreSQLBase):
+
+    def get(self, **kwargs):
+        yesterday = datetime.datetime.utcnow() - datetime.timedelta(1)
+        tomorrow = yesterday + datetime.timedelta(2)
+        yesterday = yesterday.date()
+        tomorrow = tomorrow.date()
+        filters = [
+            ('start_date', yesterday, 'date'),
+            ('end_date', tomorrow, 'date'),
+            ('product', '', 'str'),
+            ('version', '', 'str'),
+        ]
+        params = external_common.parse_arguments(filters, kwargs)
+        required = (
+            'start_date',
+            'end_date',
+            'product',
+            'version',
+        )
+        missing = []
+        for each in required:
+            if not params.get(each):
+                missing.append(each)
+        if missing:
+            raise MissingArgumentError(', '.join(missing))
+
+        SQL = """
+        SELECT
+            SUM(adi_count) AS adi_count,
+            date::DATE,
+            product_name,
+            product_os_platform,
+            product_version,
+            update_channel
+        FROM
+            raw_adi
+        WHERE
+            product_name = %(product)s AND
+            product_version = %(version)s AND
+            date BETWEEN %(start_date)s::DATE AND %(end_date)s::DATE
+        GROUP BY
+            date::DATE,
+            product_name,
+            product_os_platform,
+            product_version,
+            update_channel
+        """
+
+        results = self.query(SQL, params)
+
+        fields = (
+            'adi_count',
+            'date',
+            'product_name',
+            'product_os_platform',
+            'product_os_version',
+            'product_version',
+            'build',
+            'product_guid',
+            'update_channel',
+            'received_at',
```

Comment:
For the sake of consistency, I would like that we use the same names that are displayed in Super Search for all new services, even if that is different from what the database has. It will make it easier to for us to build new features when we're mixing Super Search with other things (and we are going to mix it with the data this ADI service exposes), and it will make the public API more consistent. 

in database         | in super search
--------------------|------------------------------
product_name        | product
product_os_platform | platform (I suppose)
product_os_version  | platform_version (I suppose)
product_version     | version
build               | build_id
product_guid        | product_id (I suppose)
update_channel      | release_channel (I suppose)